test/data/manifests/rhel_7-x86_64-azure_rhui-boot.json

```diff
@@ -22,10 +22,7 @@
       },
       {
         "name": "rhel-eng",
-        "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el7/el7-x86_64-eng-rel-20220530",
-        "package-sets": [
-          "build"
-        ]
```

Comment:
This does not look right 🤔

Reply:
hm, yeah, `gen-manifests` drops the package-sets field from repo configs.  Fixing.

Reply:
Fixed.